src/wp-includes/class-wp-script-modules.php

```diff
@@ -331,19 +331,26 @@ function ( $dependency_script_modules, $id ) use ( $import_types ) {
 	 *
 	 * @since 6.5.0
 	 *
-	 * @param array $script_module The script module.
+	 * @param string $id The script module identifier.
 	 * @return string The script module src with a version if relevant.
 	 */
-	private function get_versioned_src( array $script_module ): string {
-		$args = array();
+	private function get_src( string $id ): string {
+		if ( ! isset( $this->registered[ $id ] ) ) {
+			return '';
+		}
+
+		$script_module = $this->registered[ $id ];
+		$src           = $script_module['src'];
+
 		if ( false === $script_module['version'] ) {
-			$args['ver'] = get_bloginfo( 'version' );
+			$src = add_query_arg( 'ver', get_bloginfo( 'version' ), $src );
 		} elseif ( null !== $script_module['version'] ) {
-			$args['ver'] = $script_module['version'];
+			$src = add_query_arg( 'ver', $script_module['version'], $src );
 		}
-		if ( $args ) {
-			return add_query_arg( $args, $script_module['src'] );
-		}
-		return $script_module['src'];
+
+		/** This filter is documented in wp-includes/class-wp-scripts.php */
+		$src = apply_filters( 'script_loader_src', $src, $id );
```

Comment:
```suggestion
		$src = apply_filters( 'script_module_loader_src', $src, $id );
```

Reply:
That's frustrating :/ I guess a unique filter is workable.